packages/commonwealth/client/scripts/controllers/chain/substrate/democracy_referendum.ts

```diff
@@ -569,58 +554,4 @@ export class SubstrateDemocracyReferendum extends Proposal<
       cb
     );
   }
-
-  public unvote(who: SubstrateAccount, target?: SubstrateAccount) {
-    // you can remove someone else's vote if their unvote scope is set properly,
-    // but we don't support that in the UI right now (it requires their vote
-    // to be "expired", or for the proxy configuration to allow removing their vote)
-    if (!target) {
-      target = who;
-    }
-    return this._Chain.createTXModalData(
-      who,
-      (api: ApiPromise) =>
-        api.tx.democracy.removeOtherVote(target.address, this.data.index),
-      'unvote',
-      `${who.address} unvotes for ${target.address} on referendum ${this.data.index}`
-    );
-  }
-
-  // public async proxyVoteTx(vote: BinaryVote<SubstrateCoin>) {
-  //   const proxyFor = await (vote.account as SubstrateAccount).proxyFor.pipe(first()).toPromise();
-  //   if (!proxyFor) {
-  //     throw new Error('not a proxy');
-  //   }
-  //   const srmlVote = this._Chain.createType('Vote', {
-  //     aye: vote.choice,
-  //     conviction: convictionToSubstrate(this._Chain, weightToConviction(vote.weight)),
-  //   });
-  //   return this._Chain.createTXModalData(
-  //     vote.account as SubstrateAccount,
-  //     (api: ApiPromise) => api.tx.democracy.proxyVote(this.data.index, srmlVote),
-  //     'submitProxyDemocracyVote',
-  //     this.title
-  //   );
-  // }
-
-  public async notePreimage(author: SubstrateAccount, action: Call) {
```

Comment:
`notePreimage` and `noteImminentPreimage` still seem used in the democracy proposal form, but since we're deprecating substrate TXs entirely, we can leave it as is.

Reply:
Are we able to confirm whether check-types would have caught this?